src/api/overview.html

```diff
@@ -1,7 +1,7 @@
 <html>
 <body>
 This document is the API specification for Spring for Apache Kafka project
-<hr/>
+<hr>
```

Comment:
API doc generator was complaining about self-closing tag

Reply:
Which API doc generator? I see no errors with `$ ./gradlew clean api`.

(Thanks for the PR, BTW).

Reply:
I had this exception:

```
> Task :api
src/api/overview.html:4: error: self-closing element not allowed
<hr/>
^

> Task :api FAILED
```

If you don't have this error, I can rollback this modification.

Thanks for review.

Reply:
@artembilan Any ideas? We have this tag in all our projects.

Reply:
@hosuaby , tell us, please, what Java version do you use to build this project?
Probably there were some changes since Java 9 in the JavaDoc generator...

Reply:
Yes, I think it's because it tried to copile it with Java 13 JDK. Now I am on Java 11 and looks like everithing build just fine. I rollbacked this small modification